app/services/concerns/xccdf/rule_group_rules.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+module Xccdf
+  # Methods related to saving RuleGroupRules
+  module RuleGroupRules
+    extend ActiveSupport::Concern
+
+    included do
+      def save_rule_parents
+        @rule_group_rules = rules_with_rule_group_parent
+        ::RuleGroupRule.import!(@rule_group_rules.select(&:new_record?),
```

Comment:
Will importing only new records ever cause a conflict that you have below? Is this actually doing what we need if we only store the new stuff.

Reply:
I have a test for this where I create a `rule_group_rule` with a certain `rule_id` with a `rule_group_id` and then I call `save_rule_parents` and during this, a new record is passed with the same `rule_id` but a different `rule_group_id` and it updates the `rule_group_id` of the current `rule_group_rule` that's already in the table. So I think this is working correctly? I can't think of a use case where we'd need to look at the old stuff unless we're comparing it to something new coming in

(I just updated this test right a little to update some variable names and add a couple extra asserts so I'll push that with the other updates.)

Reply:
My point is, do we need conflict resolving below if we only import new records?

Reply:
I think for this import we might need it in case a rule's parent is updated to a different rule group, then we'd want to just update the rule's rule_group_id to the new parent rule group id instead of creating a new record

Reply:
I think that the `.select(&:new_record?)` ensures that you won't have such record :thinking: that's my problem, if you only save new records, there's nothing to overwrite.

Reply:
Just adding a comment here about what we talked about earlier. We decided to remove the conflict resolving part and just do` ignore: true` because the conflict resolving part wasn't really doing anything since we're using `.select(&:new_record?)`.  Also, we have the `rule_parent_links_to_remove` method which will remove any `rule_group_rules` that are no longer relevant.